server/src/main/java/com/google/sps/utils/AdRowProcessor.java

```diff
@@ -0,0 +1,116 @@
+/* 
+ * Description: Utility class that processes a CSV row and converts between row format and Ad format. 
+ * Author: Kira Toal
+ * Date: June 24, 2020
+ */ 
+package com.google.sps.utils;
+
+import com.google.sps.utils.Ad.Builder;
+import com.google.sps.utils.Ad;
+import java.io.IOException;
+import java.time.LocalDate;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.stream.Collectors;
+
+public final class AdRowProcessor {
```

Comment:
Why are there no tests?

Reply:
As we discussed yesterday, I will be making a new PR for the tests. I am resolving this thread to indicate that it has been addressed.